packages/core/src/Stores/notification-store.js

```diff
@@ -175,7 +175,11 @@ export default class NotificationStore extends BaseStore {
                 const sortFn = isMobile() ? sortNotificationsMobile : sortNotifications;
                 this.notification_messages = [...this.notification_messages, notification].sort(sortFn);
 
-                if (notification.key.includes('svg') || !excluded_notifications.includes(notification.key)) {
+                if (
+                    (notification.key && notification.key.includes('svg')) ||
```

Comment:
```
(notification.key?.includes('svg')) || ......
```
Can we simplify it like this?

Reply:
I didnt want to add the question mark. Instead i just check if it exists. Advice from matin